add SAML/OIDC group documentation (#8012)
cloudflare/cloudflare-docs: content/cloudflare-one/policies/access/_index.md

| Any Access Service Token | The request will need to present the headers for any [service token](/cloudflare-one/identity/service-tokens/) created for this account. |✅ | ✅ |
| Login Methods | Checks the identity provider used at the time of login. | ✅ | ❌ |
| Authentication Method | Checks the [multifactor authentication](/cloudflare-one/policies/access/mfa-requirements/) method used by the user, if supported by the identity provider. |✅ | ❌  |

> | Identity provider group| Checks the user groups (if supported) you configured with your identity provider (IdP) or LDAP with Access. The IdP group option only displays if you use a support identity provider. Supported identity providers: AzureAD, Github, Google and Okta  | ✅ | ❌ |

Reviewer: ```suggestion
| Identity provider group| Checks the user groups you configured with your identity provider (IdP). This selector only displays if you use AzureAD, Github, Google, or Okta as your IdP.  | ✅ | ❌ |
```

> | SAML Group | Checks a SAML Attribute name / value pair. This option only displays if you use a generic SAML identity provider. | ✅ | ❌ |

Reviewer: ```suggestion
| SAML Group | Checks a SAML attribute name / value pair. This selector only displays if you use a generic SAML identity provider. | ✅ | ❌ |
```
